Formula/megatools.rb

```diff
@@ -1,13 +1,13 @@
 class Megatools < Formula
   desc "Command-line client for Mega.co.nz"
   homepage "https://megatools.megous.com/"
-  url "https://megatools.megous.com/builds/megatools-1.10.3.tar.gz"
-  sha256 "8dc1ca348633fd49de7eb832b323e8dc295f1c55aefb484d30e6475218558bdb"
-  license "GPL-2.0"
+  url "https://megatools.megous.com/builds/megatools-1.11.0.20220519.tar.gz"
+  sha256 "b30b1d87d916570f7aa6d36777dd378e83215d75ea5a2c14106028b6bddc261b"
+  license "GPL-2.0-or-later"
 
   livecheck do
     url "https://megatools.megous.com/builds/"
-    regex(/href=.*?megatools[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    regex(/href=.*?megatools[._-]v?(\d{1,3}(?:\.\d{1,3})+)(?:\.\d{8})?\.t/i)
```

Comment:
cc @samford

Reply:
If the formula is going to use the full version including the date (`1.11.0.20220519`), then the `livecheck` block shouldn't be modified. The existing regex properly matches versions from filenames like `megatools-1.10.3.tar.gz` as well as `megatools-1.11.0.20220519.tar.gz` and `1.11.0.20220519` is correctly given as the newest version from livecheck.